src/poetry/factory.py

```diff
@@ -37,6 +37,14 @@ def create_poetry(
         if io is None:
             io = NullIO()
 
+        # redirect to the extended schema (from poetry, not from poetry-core)
```

Comment:
We should be updating the schema in poetry-core as well, even if the implementation is in poetry only. The intention is to drop all schemas from this repo.